src/Cool/DidYouKnow/testDidYouKnowModal.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Cool/DidYouKnow/testDidYouKnowModal.hpp"
+#include <fmt/chrono.h>
+#include <imgui.h>
+
+void did_you_know(Cool::DidYouKnowModal& _did_you_know)
```

Comment:
Our convention is to call this function `update()` as it needs to be called every frame.